ui/page/exchange/order_details_page.go

```diff
@@ -109,9 +110,14 @@ func (pg *OrderDetailsPage) HandleUserInteractions(gtx C) {
 	if pg.refreshBtn.Clicked(gtx) {
 		go func() {
 			pg.isRefreshing = true
-			pg.orderInfo, _ = pg.getOrderInfo(pg.orderInfo.UUID)
+			orderInfo, err := pg.getOrderInfo(pg.orderInfo.UUID)
+			if err != nil {
+				log.Error(err)
+				pg.notifyError(err)
+			} else {
+				pg.orderInfo = orderInfo
+			}
```

Comment:
i try to avoid if else statements when possible to enhance readability 

```
orderInfo, err := pg.getOrderInfo(pg.orderInfo.UUID)
			if err != nil {
				log.Error(err)
				pg.notifyError(err)
                               return
			}

			pg.orderInfo = orderInfo
			
```